subprojects/build-scan-performance/build-scan-performance.gradle

```diff
@@ -21,6 +21,9 @@ performanceReport {
 }
 
 dependencies {
+    // so that all Gradle features are available
+    performanceTestRuntime rootProject.configurations.testRuntime.allDependencies
```

Comment:
:o: Extract `rootProject.configuration.testRuntime.allDependencies` as something like`allTestRuntimeDependencies` and reuse?